sass/style.sass

```diff
@@ -1227,3 +1227,50 @@ textarea.deploy-key
 .content .variables-description p
 	max-width: none
 
+button.abort
```

Comment:
Can it be done with a pre-defined class instead? http://v4-alpha.getbootstrap.com/components/buttons/#sizes